### DISCUSSION TITLE
How to use CANFD with nxp,flexcan driver?

### DISCUSSION THREAD

**Discussion by tyalie** [zephyrproject-rtos/zephyr#53277] [Q&A] [answered]
Looking into the code, it seems like that CANFD is not support by the nxp flexcan driver? Are there any options to still do so?

**Comment by DerekSnell** [comment]
Hi @tyalie ,

There is currently an open PR to add CANFD support.  See https://github.com/zephyrproject-rtos/zephyr/pull/52068

Best regards

**Comment by tyalie** [comment, marked as answer]
FlexCAN CAN FD support has now successfully been merged into main with PR #56419.
